web/src/message_edit.js

```diff
@@ -471,9 +475,10 @@ function edit_message($row, raw_content) {
         }),
     );
 
-    const edit_obj = {$form, raw_content};
-    currently_editing_messages.set(message.id, edit_obj);
-    message_lists.current.show_edit_message($row, edit_obj);
+    const $message_edit_content = $form.find("textarea.message_edit_content");
+    assert($message_edit_content.length === 1);
```

Comment:
Should this just be doing a `[0]!` on the previous line rather than this assert, to get a `Jquery<HTMLElement>`?